requirements.txt

```diff
@@ -1,14 +1,43 @@
-CherryPy == 5.1.0
-djangorestframework == 3.6.2
-djangorestframework-bulk == 0.2.1
-djangorestframework-filters == 0.10.0
-django-admin-bootstrapped
-django-bootstrap3 == 8.2.2
-html5lib == 0.999
-jdcal == 1.0.1
-Markdown == 2.6.8
-openpyxl == 2.4.11
-lxml == 3.7.3
-PyJWT == 1.5.2
-requests == 2.18.4
-https://github.com/frePPLe/django/tarball/frepple_4.3
+backcall==0.1.0
+backports.functools-lru-cache==1.5
+certifi==2018.4.16
+chardet==3.0.4
+cheroot==6.2.4
```

Comment:
Cheroot is what holds CherryPy's WSGI server now, but it became separate and dependency of CherryPy around v8, so there's no point of having it along with `CherryPy==5.1.0`.
Instead, you could migrate to using pure `cheroot.wsgi.Server`.